python/ray/air/config.py

```diff
@@ -139,7 +139,7 @@ class ScalingConfig:
     """
 
     trainer_resources: Optional[Union[Dict, SampleRange]] = None
```

Comment:
Can we update the docstring for this:

```
trainer_resources: Resources to allocate for the training coordinator, which launches the worker group and executes the training code per worker while collecting reported results. This process does NOT require GPUs because it is just launching remote actors and running tasks on them. This process is always scheduled on the same node as the rank 0 worker, so one example of how this can be used is to set a minimum amount of CPU memory required on the rank 0 node. By default, this assigns 1 CPU to the training coordinator.
```